typescript-functions/main.ts

```diff
@@ -0,0 +1,54 @@
+function convertMinutesToSeconds(minutes: number): number {
+  return minutes * 60;
+}
+
+const numOfSeconds: number = convertMinutesToSeconds(5);
+console.log('Number of seconds:', numOfSeconds);
+
+function greet(name: string): string {
+  return `Hello ${name}!`;
+}
+
+const firstGreet: string = greet('Kyle');
+console.log('Greeting function:', firstGreet);
+
+const getArea = (width: number, height: number): number => {
+  return width * height;
+};
+
+const totalArea: number = getArea(5, 10);
+console.log('Total area function:', totalArea);
+
+interface Person {
+  firstName: string;
+  lastName: string;
+}
+
+const person: Person = {
+  firstName: 'Kyle',
+  lastName: 'Spence',
+};
+
+const getFirstName = (person: string): string => {
+  return person;
+};
```

Comment:
Rather than returning the whole `person` object, you should only be returning the `firstName` property of the `person` object